dbus_service.py

```diff
@@ -115,6 +117,10 @@ def __init__(
             "/StatusCode", 0
         )  # Dummy path so VRM detects us as a PV-inverter.
 
+        # add path values to dbus if servicename is inverter 
+        if self._servicename == "com.victronenergy.inverter":
+            self._dbusservice.add_path("/Mode", 2)
```

Comment:
Could you please add some documentation about what changes when the servicename is set to com.victronenergy.inverter? Best would be also to link the Victron page where the different servicenames are shortly documented.

Reply:
Will do 👍

Reply:
Very good. Please also update the readme.

Reply:
Good idea! I'll add it to it